src/main/java/io/github/apickledwalrus/skriptgui/gui/GUI.java

```diff
@@ -233,6 +265,16 @@ public Character convert(Object slot) {
 		return nextSlot();
 	}
 
+	public static InventoryClickEvent setClickedSlot(InventoryClickEvent event, int slot) {

```

Comment:
This should be moved to the GUIEvents class

Reply:
I didn't realize I only used it in GUIEvents, mb